pkg/api/latest/dynakube/certs.go

```diff
@@ -28,6 +28,7 @@ import (
 const (
 	TrustedCAKey = "certs"
 	TLSCertKey   = "server.crt"
```

Comment:
this should be renamed to ServerCertKey to free up the TLSCertKey name

Reply:
sorry, my bad - fixed now with [2eb8eab](https://github.com/Dynatrace/dynatrace-operator/pull/5589/commits/2eb8eabb3262338ff9ca007ea9abbf86447cbca8)